Package skeletons

kevin-robb opened the pull request:
Add empty catkin packages. Based on last year's codebase, except left out igvc_ros_navigation. If its contents have a good reason for being distinct from igvc_nav, it can also be created.

kevin-robb committed:
Create packages skeleton based on igvc_software_2020

kevin-robb committed:
Add maintainer and license

jkleiber commented:
Just dropping in because who doesn't love igvc_ros_navigation...

igvc_ros_navigation was just me trying to mess around with the ros navigation stack. Honestly was kinda dumb, would be more fun to make your own thing anyway

kevin-robb commented:
> Just dropping in because who doesn't love igvc_ros_navigation...
> 
> igvc_ros_navigation was just me trying to mess around with the ros navigation stack. Honestly was kinda dumb, would be more fun to make your own thing anyway

Cool cool, thanks for the info!

TylerJulian commented:
IGVC_Serial is most likely going to become IGVC_Can. I think the plan is that we aren't going to use USB to control the motor controller board anymore. 

The can-bus should handle motor control, GPS, and the remote estop.

kevin-robb committed:
Add relevant code from IGVC 2020.
Did not port over firmware, ros_navigation.
Not sure if all of igvc_serial is being replaced by CAN so I just copied it over for now in case.
